src/LoRa.h

```diff
@@ -66,7 +66,8 @@ class LoRaClass : public Stream {
   void setSyncWord(int sw);
   void enableCrc();
   void disableCrc();
-
+  void invertIQ(boolean invert);
```

Comment:
What do you think about making this:

* `enableInvertIQ()` and `disableInvertIQ()` to make it like the CRC API's above?

another option is `invertIQ()` and `noInvertIQ()`.

Reply:
Yeap, make sense InvertIQ be like CRC API's function.